src/client/app/actions/graph.ts

```diff
@@ -71,7 +71,7 @@ function shouldChangeGraphZoom(state, timeInterval) {
 
 export function changeGraphZoomIfNeeded(timeInterval) {
 	return (dispatch, getState) => {
-		if (shouldChangeGraphZoom(getState())) {
+		if (shouldChangeGraphZoom(getState(), "all")) {
```

Comment:
This is likely not correct, as it seems to cause the following error in the server:

```
GET /api/meters 200 28.757 ms - 396
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments:
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 4, _f: undefined, _strict: undefined, _locale: [object Object]
Error
    at Function.createFromInputFallback (/usr/src/app/node_modules/moment/moment.js:324:94)
    at configFromString (/usr/src/app/node_modules/moment/moment.js:2363:11)
    at configFromInput (/usr/src/app/node_modules/moment/moment.js:2589:9)
    at prepareConfig (/usr/src/app/node_modules/moment/moment.js:2572:9)
    at createFromConfig (/usr/src/app/node_modules/moment/moment.js:2539:40)
    at createLocalOrUTC (/usr/src/app/node_modules/moment/moment.js:2626:12)
    at createLocal (/usr/src/app/node_modules/moment/moment.js:2630:12)
    at hooks (/usr/src/app/node_modules/moment/moment.js:16:25)
    at stringified.split.map.timestamp (/usr/src/app/src/common/TimeInterval.js:55:82)
    at Array.map (<anonymous>)
```